### DISCUSSION TITLE
Which one is correct usage?

### DISCUSSION THREAD

**Discussion by mertasan** [alpinejs/alpine#1581] [1. Help]
Which one is correct usage?

```html
<div 
    x-show="isUploading"
    x-transition.opacity
    x-transition.duration.150ms
>
```

```html
<div 
    x-show="isUploading"
    x-transition.opacity.duration.150ms
>
```

**Edit**: I found an example here. Thanks.

https://github.com/laravel/jetstream/blob/28b235c6c6c0ca15987bdc9873ea05a37dce6691/resources/views/components/action-message.blade.php#L6

**Comment by devcircus** [comment]
It looks like to me, according to the [docs](https://alpinejs.dev/directives/transition), either way is correct. The modifiers can be added to the end of the transition type directive or can be in their own directive. If not, it probably needs to be more clear in the docs.
